docs/best_practices/tool_xml.rst

```diff
@@ -88,6 +88,14 @@ would have a hard time finding it and gathering context about its functionality.
 With the VSearch prefix, once a user learns what one VSearch tool does, they can
 quickly apply that to the other available VSearch tools.
 
+Tool profile
+------------
+
+Tools should define a recent profile:
+
+- for new tools IUC requires the most recent Galaxy release
```

Comment:
Is that not be too strict? I guess most Galaxy instances are lacking 1-2 version behind.

Reply:
That seems reasonable to me, a little leeway.

Reply:
I changed it to 1 year, i.e. 3 versions and used "should".